config/captcha.php

```diff
@@ -1,5 +1,30 @@
 <?php
-return [    
+return [
     'app.recaptcha.key' => env("GOOGLE_RECAPTCHA_SITEKEY", null),
     'app.recaptcha.secret' => env("GOOGLE_RECAPTCHA_SECRET", null),
-];
\ No newline at end of file
+
+    /**
+     * Configuração para implementação de captcha seguindo um novo padrão de configuração
+     * 
+     * - O exemplo abaixo segue o padrão de configuração para o Google Recaptcha atualmente utilizado
+     * - Esse padrão de configuração permite a implementação de outros provedores de captcha, como por exemplo o Cloudflare Turnstile
+     */
+    'captcha' => [
+        'provider' => 'cloudflare',
+        // 'provider' => 'google',
```

Comment:
Sugiro evitar definição rígida de `provider` como `cloudflare`. Ao invés disso, verificar a existência de uma variável de ambiente `CAPTCHA_PROVIDER` e, se ela não existir, assumir o valor `google` a fim de manter retrocompatibilidade.
```suggestion
        'provider' => env('CAPTCHA_PROVIDER', 'google'),
```